lib/cocoapods-search/command/search.rb

```diff
@@ -82,7 +84,19 @@ def local_search
           sets.reject! { |set| !set.specification.available_platforms.map(&:name).include?(platform) }
         end
 
-        sets.reverse_each do |set|
+        if use_pager
+          extend_ui_puts
+          IO.popen('$PAGER', 'w') do |io|
```

Comment:
I think we should resolve this ourselves and fallback to less/more in the case where a user doesn't have an explicit `PAGER` set?